coordinator/Cargo.toml

```diff
@@ -4,124 +4,56 @@ version = "1.8.11"
 edition = "2021"
 
 [dependencies]
+anyhow = { version = "1", features = ["backtrace"] }
 atty = "0.2.14"
-bitcoin = "0.29.2"
+axum = { version = "0.6.20", features = ["ws", "query", "multipart"] }
+bdk = { version = "1.0.0-alpha.6", features = ["std"] }
+bdk_file_store = "0.6"
+bitcoin = { version = "0.29.2" }
+bitcoin_30 = { package = "bitcoin", version = "0.30" }
```

Comment:
It seemed easier to approach this work by naming the new dependency like this, but perhaps now that it compiles I can go with `bitcoin` for version 0.30 and `bitcoin_old` for the old version?